libraries/engage/core/actions/requestAppPermission.js

```diff
@@ -0,0 +1,26 @@
+import { requestAppPermissions } from '@shopgate/pwa-core/commands/appPermissions';
+import { PERMISSION_STATUS_NOT_SUPPORTED } from '@shopgate/engage/core/constants';
+import { appPermissionStatusReceived } from '../action-creators';
+
+/**
+ * Requests an additional app permission from the operating system.
+ * Additionally it propagates * the received status via the APP_PERMISSION_STATUS_RECEIVED action.
```

Comment:
What does the * mean?

Reply:
leftover from re-formatting the jsDoc - updated